### ISSUE TITLE
Scroll position (offset) is lost when viewing the details of an item in the wizard

### ISSUE THREAD

**Issue by mrlacey** [microsoft/WindowsTemplateStudio#3804]
**Describe the bug**
When viewing the details of an item in the wizard and then returning to the list, the wizard automatically scrolls back to the top of the list. 
Because of this:
- it can be easy to lose context of where you are in the wizard and what is being viewed.
- it forces unnecessary extra scrolling to return to where you were in the list if you want to select the item that was just viewed or if you want to see other items further down the list from what was just viewed.
- If navigating by keyboard (or anything other than the mouse) the experience is particularly bad as the focus is moved to elsewhere in the wizard.

**To Reproduce**
1. In the new UWP project wizard
2. Click on pages
3. Scroll down to 'Ink Draw'
4. Click on 'Details'
5. Click Back arrow
6. See list scroll back to the top.

(gif below shows executing these steps)

**Expected behavior**
When navigating back, the scroll position of the list should be maintained and focus should be on the item just viewed.

**Screenshots**
![winTs-wizard-scroll-issue](https://user-images.githubusercontent.com/189547/89718045-08035c00-d9b4-11ea-9814-5813a7c96bcc.gif)

_In the above notice that after clicking '<- Back' the list scrolls back to the top before the gif repeats_

**Additional context**
Applies to UWP and WPF but only UWP has sufficient options to make this visible.

Applies to the following platforms:
- [x] UWP
- [x] WPF

**Comment by sibille** [comment]
Verified in dev-nightly: 
Templates version: 0.22.20262.1
Wizard version: 0.22.20262.1
